src/main/scala/io/shiftleft/js2cpg/preprocessing/TypescriptTranspiler.scala

```diff
@@ -91,7 +91,10 @@ class TypescriptTranspiler(override val config: Config, override val projectPath
     val command = if (pnpmAvailable(projectPath)) {
       s"${TranspilingEnvironment.PNPM_ADD} typescript"
     } else if (yarnAvailable()) {
-      s"${TranspilingEnvironment.YARN_ADD} typescript"
+      if (logger.isDebugEnabled)
+        s"${TranspilingEnvironment.YARN_ADD} typescript"
+      else
+        s"${TranspilingEnvironment.YARN_ADD} -v typescript"
```

Comment:
I think this is the wrong way around, isn't it?
Debug is `-v`?

Reply:
:facepalm: yes